### PR TITLE
(#214) Added LDAP Setup Documentation to the CCM Web Documentation.

### PR DESCRIPTION
**Pull request by ryanrichter94**
Created LDAP setup documentation for CCM in CCM web documentation.

### CODE REVIEW

File: CentralManagement/CentralManagementSetupWeb.md [hunk 1]
```diff
@@ -7,47 +7,7 @@ This is the Central Management website that gives an API and a web layer to cent
...
... FilePath](#cannot-process-command-because-of-one-or-more-missing-mandatory-parameters-filepath)auto    - [All attempts to send email from CCM result in an error](#all-attempts-to-send-email-from-ccm-result-in-an-error)auto    - [Emails sent from CCM to users has links that contains localhost, rather than actual CCM Server name](#emails-sent-from-ccm-to-users-has-links-that-contains-localhost-rather-than-actual-ccm-server-name)auto    - [The updated license file is not being picked up in the website](#the-updated-license-file-is-not-being-picked-up-in-the-website)auto    - [A computer or group is not showing as available for deployments but I have plenty of available licenses](#a-computer-or-group-is-not-showing-as-available-for-deployments-but-i-have-plenty-of-available-licenses)auto    - [Failed to generate a user instance of SQL Server due to failure in retrieving the user's local application data path.](#failed-to-generate-a-user-instance-of-sql-server-due-to-failure-in-retrieving-the-users-local-application-data-path)auto    - [The website app pool shuts down almost as soon as it is started](#the-website-app-pool-shuts-down-almost-as-soon-as-it-is-started)autoauto<!-- /TOC -->
```
Comment (ferventcoder) [inline CentralManagement/CentralManagementSetupWeb.md:10]: Go into your VS Code settings and add `"files.eol": "\n"`. If it is set to "auto", this is what you get in the TOC.

**Review by ferventcoder — CHANGES_REQUESTED** [review]
Fix the TOC bit

File: CentralManagement/CentralManagementSetupWeb.md [hunk 2]
```diff
@@ -247,6 +208,22 @@ You should received a notification similar to this:
...
+1. Login with the `ccmadmin` user
+1. In the left hand menu click on `Administration` and then `Settings`
+1. Click on the `User management` tab in the `Settings` screen
+1. Under LDAP Setting click the `Enable LDAP Authentication` button
+1. Fill in your FQDN for the `Domain name` field
+1. Fill in the `User name` field with an active directory account that has access to query user accounts within your active directory enviornment.
+1. Fill the `Password` field with the password for the active driectory user name used above
+1. Click the `Save All` button at the top right of the page to save your settings.
+
+![CCM LDAP Setup](images/features/ccm/ccm_ldap_setup.png)
```
Comment (ferventcoder) [inline CentralManagement/CentralManagementSetupWeb.md:225]: Zoom in a bit

File: CentralManagement/CentralManagementSetupWeb.md [hunk 3]
```diff
@@ -247,6 +208,22 @@ You should received a notification similar to this:
...
+
+CCM is able to bind with LDAP to enable authentication to Central Management using an LDAP user account and password.
+
+1. Open the CCM Site in the browser
+1. Login with the `ccmadmin` user
+1. In the left hand menu click on `Administration` and then `Settings`
+1. Click on the `User management` tab in the `Settings` screen
+1. Under LDAP Setting click the `Enable LDAP Authentication` button
+1. Fill in your FQDN for the `Domain name` field
+1. Fill in the `User name` field with an active directory account that has access to query user accounts within your active directory enviornment.
```
Comment (ferventcoder) [inline CentralManagement/CentralManagementSetupWeb.md:221]: typo on environment.

File: CentralManagement/CentralManagementSetupWeb.md [hunk 4]
```diff
@@ -247,6 +208,22 @@ You should received a notification similar to this:
...
+1. Fill the `Password` field with the password for the active driectory user name used above
+1. Click the `Save All` button at the top right of the page to save your settings.
```
Comment (ferventcoder) [inline CentralManagement/CentralManagementSetupWeb.md:223]: Be consistent with periods or not at the end of each list item.

**Comment by ferventcoder** [comment]
Just a few more things to fix up. Pretty good otherwise!

File: CentralManagement/CentralManagementSetupWeb.md (hunk 1 above)
Comment (ryanrichter94) [reply CentralManagement/CentralManagementSetupWeb.md:10]: Made VS Code setting changes to get TOC working correctly.

File: CentralManagement/CentralManagementSetupWeb.md (hunk 2 above)
Comment (ryanrichter94) [reply CentralManagement/CentralManagementSetupWeb.md:231]: Added new screen shot

File: CentralManagement/CentralManagementSetupWeb.md (hunk 3 above)
Comment (ryanrichter94) [reply CentralManagement/CentralManagementSetupWeb.md]: Learned to spell correctly.

File: CentralManagement/CentralManagementSetupWeb.md (hunk 4 above)
Comment (ryanrichter94) [reply CentralManagement/CentralManagementSetupWeb.md:223]: Added periods to end of all list items for document continuity.

**Comment by gep13** [comment]
@ryanrichter94 seems like the force push that has been done here doesn't have the changes that you mentioned were addressed.  Let me know if you want to grab some time at some point to walk through what is being done here.

**Comment by steviecoaster** [comment]
I've reviewed this, and all changes requested have been addressed, and it looks good. I'm going to merge this in.

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "files.eol": "\n"
+}
```
Comment (ferventcoder) [inline .vscode/settings.json:3]: Well that's interesting. Did not know this was something that we'd want to do. No biggie.

File: CentralManagement/CentralManagementSetupWeb.md [hunk 6]
```diff
@@ -247,6 +248,32 @@ You should received a notification similar to this:
 
 ![Test email sent successfully](images/features/ccm/test_email_sent_correctly.png)
 
+### Step 4.3: LDAP Configuration
+
```
Comment (ferventcoder) [inline CentralManagement/CentralManagementSetupWeb.md:252]: It's probably worth a revisit to mention this is an optional configuration step.

**Pull request merged**